test/member_data_test.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/members_page.rb'
+
+describe 'member data' do
+  url = 'http://www.parliament.go.tz/mps-list'
+  around { |test| VCR.use_cassette(url.split('/').last, &test) }
+
+  subject do
+    MembersPage.new(response: Scraped::Request.new(url: url).response)
+               .member_rows
+               .select { |row| row.to_h[:id] == id }.first
```

Comment:
As with Sint Maarten: `.select { … }.first` should be `.find`, and `row.to_h[:id]` could just be `row.id`